### PR TITLE
Users' guide in .rst format

### PR DESCRIPTION
**Pull request by bradlaufenberg**
@gonuke These are the first three files I have transferred to restructured text formats. I am just using these first three files as practice for pushing my progress to git and creating a pull request.

### CODE REVIEW

File: doc/introtext.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+===================
+
+Analytic and Laplacian Adaptive Radioactivity Analysis
+======================================================
+--------------
+
+INTRODUCTION:
+=============
+
+ The primary purpose of ALARA is to calculate the induced activation caused by neutron irradiation throughout a nuclear system (including fission reactors,
```
Comment (gonuke) [inline doc/introtext.rst]: For long lines, it is useful to force a "hard line wrap" with only about 60 characters per line.  This ensures that when there are future modifications, they are more localized and easier to review.

Comment (gonuke) [reply doc/introtext.rst]: The text editor you use may have features to help with that.

**Comment by bradlaufenberg** [comment]
These are the rest of the pages for the Users' guide besides the input syntax page. I have also edited the other documents slightly by adding hyperlinks to other parts of the users' guide (i.e. the glossary).

**Comment by bradlaufenberg** [comment]
Closed this PR because PR #22 includes all of these commits, the ones made by @gonuke and more from myself.

**Pull request closed without merging**